src/test/java/org/gridsuite/study/server/utils/ImpactUtils.java

```diff
@@ -25,13 +25,15 @@ private ImpactUtils() {
 
     public static Optional<NetworkModificationResult> createModificationResultWithElementImpact(SimpleImpactType impactType, IdentifiableType elementType, String elementId, Set<String> substationIds) {
         return Optional.of(NetworkModificationResult.builder()
-            .networkImpacts(List.of(createElementImpact(impactType, elementType, elementId, substationIds)))
-            .build());
+                .networkImpacts(List.of(createElementImpact(impactType, elementType, elementId, substationIds)))
+                .applicationStatus(NetworkModificationResult.ApplicationStatus.ALL_OK)
+                .build());
     }
 
     public static Optional<NetworkModificationResult> createModificationResultWithElementImpact(SimpleElementImpact impact) {
         return Optional.of(NetworkModificationResult.builder()
             .networkImpacts(List.of(impact))
+            .applicationStatus(NetworkModificationResult.ApplicationStatus.ALL_OK)
```

Comment:
Not really needed, with Builder.Default annotation on applicationStatus field

Reply:
Indeed I forgot to remove it after adding Builder.Default